changelog/2830.doc.rst

```diff
@@ -0,0 +1 @@
+Added the |force-free| definition to the |glossary|.(:pr:`2323`)
```

Comment:
```suggestion
Added the :term:`force-free` definition to the |glossary|.
```
The `|force-free|` syntax requires that we define a [global substitution](https://docs.plasmapy.org/en/latest/contributing/doc_guide.html#substitutions), and the link to the pull request will get automatically added when the pull request documentation is built.

Reply:
Thanks! I did not understand very well how they work